site/docs/voice/bxml/startStream.mdx

```diff
@@ -0,0 +1,419 @@
+---
+id: startStream
+title: Start Stream
+slug: /voice/bxml/startStream
+description: A general overview of Bandwidth's StartStream BXML Verb
+keywords:
+  - bandwidth
+  - voice
+  - bxml
+  - start
+  - streaming
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The StartStream verb allows a segment of a call to be sent off to another destination for additional processing.
+
+When used on a call, audio from one or both sides (tracks) of the call will be sent to the specified destination. The stream will continue until the call ends or the [`<StopStream>`][1] verb is used. A total of 4 concurrent track streams are allowed on a call. A `<StartStream>` request that uses `both` tracks will count as 2 concurrent streams.
+A call has only two tracks, named after the direction of the media from Programmable Voice perspective:
```

Comment:
```suggestion
A call has only two tracks, which are named after the direction of the media from the perspective of the Programmable Voice platform:
```

Reply:
Alternatively, instead of `Programmable Voice platform`, we could say `Bandwidth` or `us` or `this API`.